src/kontrol/kdist/cheatcodes.md

```diff
@@ -1171,7 +1172,7 @@ Will also return true if REASON is `.Bytes`.
     syntax Bool ::= "#matchReason" "(" Bytes "," Bytes ")" [function, klabel(foundry_matchReason)]
  // ----------------------------------------------------------------------------------------------
     rule #matchReason(REASON, _) => true requires REASON ==K .Bytes
-    rule #matchReason(REASON, OUT) => REASON ==K #range(OUT, 4, lengthBytes(OUT) -Int 4) requires REASON =/=K .Bytes
+    rule #matchReason(REASON, OUT) => #range(REASON, lengthBytes(REASON) -Int 32, 32) ==K #range(OUT, lengthBytes(OUT) -Int 32, 32) requires REASON =/=K .Bytes
```

Comment:
Before approving, could you double check that this the same behaviour that forge has for `expectRevert` ? [link to rust implementation](https://github.com/foundry-rs/foundry/blob/master/crates/cheatcodes/src/test/expect.rs#L494-L546)